pkg/importer/vddk-datasource.go

```diff
@@ -247,19 +247,13 @@ func createVMwareClient(endpoint string, accessKey string, secKey string, thumbp
 		return nil, err
 	}
 
-	// Construct VMware SDK URL and get MOref
-	sdkURL := vmwURL.Scheme + "://" + accessKey + ":" + secKey + "@" + vmwURL.Host + "/sdk"
-	vmwURL, err = url.Parse(sdkURL)
-	if err != nil {
-		klog.Errorf("Unable to create VMware URL: %v", err)
-		return nil, err
-	}
+	vmwURL.User = url.UserPassword(accessKey, secKey)
+	vmwURL.Path = "sdk"
 
 	// Log in to vCenter
 	ctx, cancel := context.WithCancel(context.Background())
 	conn, err := govmomi.NewClient(ctx, vmwURL, true)
 	if err != nil {
-		klog.Errorf("Unable to connect to vCenter: %v", err)
```

Comment:
I removed it because there are already a few other places where the error is logged, so it piles up in an unhelpfully verbose way without adding much new information. I think there are a few other places in the VDDK importer that should just return an error instead of logging it separately and then returning it.

I left it removed after taking out the previous redaction code, but the change does look out of place without that context. I can put it back if it's tidier that way.

Reply:
thanks, if the log is too chatty, better have another PR to fix that.

It does not worth much, but this lgtm.